Source/Csla.Channels.RabbitMq/RabbitMqProxy.cs

```diff
@@ -58,13 +58,18 @@ public RabbitMqProxy(ApplicationContext applicationContext, RabbitMqProxyOptions
     /// </summary>
     private ProxyListener? QueueListener { get; set; }
 
+    /// <summary>
+    /// Gets the proxy options.
+    /// </summary>
+    public RabbitMqProxyOptions Options { get; }
+
     /// <summary>
     /// Method responsible for creating the Connection,
     /// Channel, ReplyQueue, and DataPortalQueueName values
     /// used for bi-directional communication.
     /// </summary>
 #if NET8_0_OR_GREATER
-    [MemberNotNull(nameof(Connection), nameof(Channel), nameof(QueueListener))]
+      [MemberNotNull(nameof(Connection), nameof(Channel), nameof(QueueListener))]
```

Comment:
Why adding the spaces here? Shouldn't it be on the same level as the method and xml-doc?